octavia-cli/octavia_cli/entrypoint.py

```diff
@@ -2,41 +2,53 @@
 # Copyright (c) 2021 Airbyte, Inc., all rights reserved.
 #
 
+import airbyte_api_client
 import click
+from airbyte_api_client.api import workspace_api
 
 
 @click.group()
 @click.option("--airbyte-url", envvar="AIRBYTE_URL", default="http://localhost:8000", help="The URL of your Airbyte instance.")
-def octavia(airbyte_url):
-    # TODO: check if the airbyte_url is reachable
-    click.secho(f"🐙 - Octavia is targetting your Airbyte instance running at {airbyte_url}")
+@click.pass_context
+def octavia(ctx: click.Context, airbyte_url: str) -> None:
+    ctx.ensure_object(dict)
+    client_configuration = airbyte_api_client.Configuration(host=f"{airbyte_url}/api")
+    api_client = airbyte_api_client.ApiClient(client_configuration)
+    # TODO alafanechere workspace check might deserve its own function
+    api_instance = workspace_api.WorkspaceApi(api_client)
+    api_response = api_instance.list_workspaces()
+    # TODO alafanechere prompt user to chose a workspace if multiple workspaces exist
+    workspace_id = api_response.workspaces[0].workspace_id
```

Comment:
Will it also be able to specify the workspace within a config file directly in the future (to avoid choosing either the default one or to be prompted to choose one)?

Reply:
Yes, we could:
1. parse an environment variable such as `AIRBYTE_WORKSPACE_ID`
2. validate the workspace exists on the targeted airbyte URL
3. use this workspace id in the command